Source/FakeItEasy/Creation/DefaultFakeCreatorFacade.cs

```diff
@@ -126,6 +126,11 @@ public IFakeOptionsBuilder<T> Implements(Type interfaceType)
                 return this;
             }
 
+            public IFakeOptionsBuilder<T> Implements<TInterface>()
+            {
+                return Implements(typeof (TInterface));

```

Comment:
Oh, an this is even more minor, but an additional warning from the compiler asks for `typeof(TInterface` instead of `typeof (TInterface`. Again, this is a "nice-to-have".